### ISSUE TITLE
[🐛 Bug]: org.openqa.selenium.grid.server not releasing port even after stopping it

### ISSUE THREAD

**Issue by sathanRaj** [SeleniumHQ/selenium#14901]
### What happened?

I'm using selenium 4.24.0 version. In my case i need to stop the selenium server and start it again with the previous port.
but, when i stop the server it's not releasing the port.So I'm not able to use the same port and I need to stop the whole java process to free the port.

### How can we reproduce the issue?

```shell
Here is the code snippet:
Code used to start the server
                        Hub hub = new Hub();
			Server hubServer = hub.asServer("Here is my custom config");
			hubServer.start();
Code used to stop the server
                        hubServer.stop();
```

### Relevant log output

```shell
getting below error when i try to start the server with the same port

Caused by: org.zeromq.ZMQException: Errno 48 : Address already in use
	at org.zeromq.ZMQ$Socket.mayRaise(ZMQ.java:3901)
	at org.zeromq.ZMQ$Socket.bind(ZMQ.java:3176)
	at org.openqa.selenium.events.zeromq.BoundZmqEventBus.<init>(BoundZmqEventBus.java:56)
	at org.openqa.selenium.events.zeromq.ZeroMqEventBus.create(ZeroMqEventBus.java:49)
	at org.openqa.selenium.events.zeromq.ZeroMqEventBus.create(ZeroMqEventBus.java:91)
	... 27 more
```

### Operating System

Mac

### Selenium version

4.24.0

### What are the browser(s) and version(s) where you see this issue?

It's not browser related

### What are the browser driver(s) and version(s) where you see this issue?

It's not browser related

### Are you using Selenium Grid?

4.24.0

**Comment by joerg1985** [comment]
@sathanRaj this should have been fixed in 4.27.0 with https://github.com/SeleniumHQ/selenium/commit/03386772307dde5dd063ef1bde35d8013132df94 could you test with this version?

**Comment by sathanRaj** [comment]
Working as expected in 4.27.0 version.
Thanks for the info @joerg1985
